src/components/Form.js

```diff
@@ -16,16 +17,17 @@ const Addform = () => {
   };
   return (
     <>
-      <h2>Add New Book</h2>
-      <form onSubmit={addNewBook}>
-        <input type="text" name="bookTitle" placeholder="Booktitle" />
-        <input type="text" name="author" placeholder="Author" />
-        <select name="category">
+      <h3 className={styles.headerform}>ADD NEW BOOK</h3>
```

Comment:
- [ ] Same here. The color of the text is not correct as per the given design. It should be `#888888`. Kindly fix this issue.

Reply:
This change has been done well